src/android/ServerSyncUtil.java

```diff
@@ -0,0 +1,19 @@
+package edu.berkeley.eecs.emission.cordova.serversync;
+
+import static edu.berkeley.eecs.emission.cordova.serversync.ServerSyncPlugin.AUTHORITY;
```

Comment:
we need to either remove this import or add it to the hook.

Reply:
Actually, since this is defined in the plugin and not the main package, it works fine.
We only need to rewrite for the files that are generated in the main package.